src/map/packets.hpp

```diff
@@ -356,6 +356,38 @@ struct PACKET_ZC_ACK_OPEN_BANKING{
 } __attribute__((packed));
 DEFINE_PACKET_HEADER(ZC_ACK_OPEN_BANKING, 0x9b7)
 
+#if PACKETVER > 6
+struct PACKET_ZC_REQ_EXCHANGE_ITEM{
+	uint16 packetType;
+	char requesterName[NAME_LENGTH];
+	uint32 targetId;
+	uint16 targetLv;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_REQ_EXCHANGE_ITEM, 0x1f4)
+#else
+struct PACKET_ZC_REQ_EXCHANGE_ITEM{
+	uint16 packetType;
+	char requesterName[NAME_LENGTH];
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_REQ_EXCHANGE_ITEM, 0x9a)
+#endif
+
+#if PACKETVER > 6
+struct PACKET_ZC_ACK_EXCHANGE_ITEM{
+	uint16 packetType;
+	uint8 result;
+	uint32 targetId;
+	uint16 targetLv;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_ACK_EXCHANGE_ITEM, 0x1f5)
+#else
+struct PACKET_ZC_ACK_EXCHANGE_ITEM{
+	uint16 packetType;
+	uint8 result;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_ACK_EXCHANGE_ITEM, 0xe7)
```

Comment:
This change does not match the previous behavior.

https://github.com/rathena/rathena/pull/8581/files#diff-e095af7d5069bf7fbe9926283903389918f6afe47ea6bae5a4880503b59c7d2cL4696

0xe7 was sent when tsd is nullptr for PACKETVER > 6

Reply:
same to:  https://github.com/rathena/rathena/pull/8581/files#r1736556896